sql/hive-thriftserver/pom.xml

```diff
@@ -77,6 +77,10 @@
       <groupId>${hive.group}</groupId>
       <artifactId>hive-beeline</artifactId>
     </dependency>
+    <dependency>
+      <groupId>${hive.group}</groupId>
+      <artifactId>hive-service</artifactId>
```

Comment:
> Or is it that we now depend on it just so that Hive JDBC client can work?

yes, beeline rely on hive-beeline -> hive-jdbc -> hive-service. 
So I just import this part for beeline, and this part is follow hive version. 
So it is ok to do this.